tests/playwright/specs/user.js

```diff
@@ -0,0 +1,25 @@
+const { CreateAccountPage } = require( '../pageobjects/index.js' );
```

Comment:
index.js is not required here.

Reply:
@vidhi-mody, eslint expects the import to be /index.js. There is an eslint error when I remove the index.js part.

Reply:
It has been fixed, added eslint lint rule.

Reply:
I have added eslint ignore rule. It should be fine now